esmvalcore/_version.py

```diff
@@ -1,2 +1,7 @@
-"""ESMValCore version."""
-__version__ = '2.5.0'
+from importlib.metadata import PackageNotFoundError, version
+
+try:
+    __version__ = version("ESMValCore")
```

Comment:
It works and the package name is ESMValCore
https://github.com/ESMValGroup/ESMValCore/blob/8af46d466f94f4441778ed612b3b8393f7d463d8/setup.py#L194

Reply:
Lowercase works too though, so I think the command is case insensitive. Probably makes sense, given that Python works on Windows too.

Reply:
Nice. I'm not sure what magic is happening because in other cases things are case sensitive (question for another day)

```sh
In [1]: import ESMValCore
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import ESMValCore

ModuleNotFoundError: No module named 'ESMValCore'

In [2]: import esmvalcore
```